common/api/presentation-common.api.md

```diff
@@ -1827,6 +1833,7 @@ export interface PresentationRpcResponseData<TResult = undefined> {
 
 // @public
 export enum PresentationStatus {
+    // @deprecated
     BackendTimeout = 65543,
     Canceled = 1,
     Error = 65536,
```

Comment:
Should this throw a runtime error (and result in HTTP status 500) instead?

Reply:
Can you clarify what you mean by "this"? `PresentationStatus.BackendTimeout` is not used anymore.

Reply:
`PresentationStatus.Error` (the last line in the snippet is the one I submitted the comment on)

Reply:
For errors that are likely bugs on our end, we throw generic `Error`, which ends up as a HTTP 500 response. 

For errors that are user input-related (e.g. your presentation rules contain a non-existing class), we throw `PresentationError` with `PresentationStatus.Error`, which ends up as HTTP 422 response.

I don't think we want to throw 5xx errors for something that's not server-related.